setup.py

```diff
@@ -10,7 +10,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/c4deszes/ldfparser",
-    packages=find_packages(),
+    packages=find_packages(exclude=["tests"]),
```

Comment:
The root cause of tests being included is the fact that there's an `__init__.py` file in the tests directory. It's not intentional and it's not recommended by `pytest` but imports might break if we delete that file so I'd accept this solution.